rs/boundary_node/ic_boundary/src/core.rs

```diff
@@ -345,6 +374,84 @@ pub async fn main(cli: Cli) -> Result<(), Error> {
         vec![Box::new(metrics_runner), Box::new(generic_limiter_runner)];
     runners.append(&mut registry_runners);
 
+    // HTTP Logs Anonymization
+    let tracker = (cli.misc.crypto_config)
+        .zip(cli.obs.obs_log_anonymization_canister_id)
+        .map(|(crypto_config, log_anonymization_cid)| {
+            #[allow(clippy::disallowed_methods)]
+            let c = tokio::task::block_in_place(|| {
```

Comment:
Given that `block_in_place` is a disallowed method in our repo, even though this is only executed once as part of `main`, I'd still avoid it, so as to set a good example, given that this is easily possible:
```
let tracker = if let (Some(crypto_config), Some(log_anonymization_cid)) = (
        cli.misc.crypto_config,
        cli.obs.obs_log_anonymization_canister_id,
    ) {
        let registry_client_clone = registry_client.clone();
        let c = tokio::task::spawn_blocking(move || {
            Arc::new(CryptoComponent::new(
                &crypto_config,                          // config
                Some(tokio::runtime::Handle::current()), // tokio_runtime_handle
                registry_client_clone,                   // registry_client
                no_op_logger(),                          // logger
                None,                                    // metrics
            ))
        })
        .await
        .expect("Failed to await JoinHandle");

        let pk = c
            .current_node_public_keys()
            .map_err(|err| anyhow!("failed to retrieve public key: {err:?}"))?
            .node_signing_public_key
            .context("missing node public key")?;

        let nid = derive_node_id(&pk).expect("failed to derive node id");

        // Custom Signer
        let s = Sender::Node {
            pub_key: pk.key_value,
            sign: Arc::new(move |msg: &MessageId| {
                #[allow(clippy::disallowed_methods)]
                let sig = tokio::task::block_in_place(|| {
                    c.sign_basic(
                        msg,                                  // message
                        nid,                                  // signer
                        registry_client.get_latest_version(), // registry_version
                    )
                    .map(|value| value.get().0)
                    .map_err(|err| anyhow!("failed to sign message: {err:?}"))
                })?;

                Ok(sig)
            }),
        };

        // Agent
        let (proto, port) = [
            // HTTP
            cli.listen.listen_http_port.map(|p| ("http", p)),
            // HTTPS
            #[cfg(feature = "tls")]
            cli.listen.listen_https_port.map(|p| ("https", p)),
        ]
        .into_iter()
        // Choose first non-none option
        .flatten()
        .next()
        .ok_or(anyhow!("missing http and https listen ports"))?;

        let ag = Agent::new(
            format!("{proto}://localhost:{port}").parse()?, // url
            s,                                              // sender
        );

        // Canister
        let c = AnonymizationCanister::new(
            ag,                    // agent
            log_anonymization_cid, // canister_id
        );

        // Rng
        let rng_seed = SystemTime::now().duration_since(UNIX_EPOCH)?.as_nanos();
        let rng = Box::new(StdRng::seed_from_u64(rng_seed as u64));

        // Tracker
        let tracker = AnonymizationTracker::new(rng, c.into())?;

        Some(tracker)
    } else {
        None
    };
```

Reply:
This change should be made now, thanks for the help and advice.